.github/workflows/version.yaml

```diff
@@ -0,0 +1,84 @@
+name: Check Package Versions
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  check-versions:
+    runs-on: token-studio-linux-medium
+    container:
+      image: node:20.17.0-alpine3.20 # Using alpine for a smaller image
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@a5ac7e51b41094c92402da3b24376905380afc29
```

Comment:
Add specific version as # after the commit hash.

Reply:
version added